src/Pages/AccountOverview.tsx

```diff
@@ -0,0 +1,107 @@
+    const { accountInformation, } = useApplicationState();
+    const [accountBalances, setAccountBalances] = useState<AccountBalanceRecord[]>([]);
+    const [accountCreated, setAccountCreated] = useState<{date?: string, by?: string}>({})
+
+    useEffect(() => {
+
+        if (accountInformation.account) {
+            new TransactionCallBuilder(new URI(horizonUrl()))
+                .forAccount(accountInformation.account.id)
+                .order('asc')
+                .limit(1)
+                .call()
+                .then(({records}) => records.pop())
+                .then(record => record?.operations())
+                .then(operations => operations?.records.find(op => op.type === Horizon.OperationResponseType.createAccount) as ServerApi.CreateAccountOperationRecord)
+                .then((createAccountOperation: ServerApi.CreateAccountOperationRecord) => {
+                    setAccountCreated({by: createAccountOperation.funder, date: createAccountOperation.created_at});
+                });
+
+            setAccountBalances(accountInformation.account?.balances.map(
+                (balanceLine) => {
+                    let reserves = new BigNumber(0);
+                    if (balanceLine.asset_type === 'native') {
+                        const subentryCount = new BigNumber(accountInformation.account?.subentry_count??0);
+                        reserves = reserves
+                            // reserves for data-entries, offers, signers, trust-lines
+                            .add(subentryCount.isZero() ? 0 : subentryCount.div(2))
+                            // account base reserve
+                            .add(1);
+                    }
+
+                    return {
+                        asset: balanceLine.asset_type !== 'native'
+                            ? `${balanceLine.asset_code}:${balanceLine.asset_issuer}`
+                            : 'native:XLM',
+                        balance: new BigNumber(balanceLine.balance),
+                        buyingLiabilities: new BigNumber(balanceLine.buying_liabilities),
+                        sellingLiabilities: new BigNumber(balanceLine.selling_liabilities),
+                        spendable: new BigNumber(balanceLine.balance)
+                            .sub(new BigNumber(balanceLine.selling_liabilities))
+                            .sub(reserves)
+                    };
+                }
+            ));
+        }
+        // eslint-disable-next-line
+    }, [accountInformation.account]);
+
+    return (<>
+        <AccountSelector />
+        <>Account {shortAddress(accountInformation.account?.id??'', 9)} created on {accountCreated.date} by <Link to={'/account/'+accountCreated.by}>{shortAddress(accountCreated.by??'', 9)}</Link></>
+        <Table
+            showHeader={false}
+            columns={balancesTableColumns()}
+            dataSource={accountBalances}
+            rowKey='asset'
+            pagination={false}
+        />
```

Comment:
display all the accounts balances. rendering is [defined above](https://github.com/hanseartic/stellar-claim/pull/2/files#r688115439).